docs/modeling/compound-models.rst

```diff
@@ -287,7 +287,7 @@ combination of classes *and* instances in the same expression::
         Outputs: ('y',)
         Fittable parameters: ('slope', 'intercept')
     <BLANKLINE>
-        [1]: <Sine1D(amplitude=1.0, frequency=1.0)>
+        [1]: <Sine1D(amplitude=1.0, frequency=1.0, phase=0.0)>
```

Comment:
The line above which reads:
```
Fittable parameters: ('slope_0', 'intercept_0', 'amplitude_1', 'frequency_1')
```
should be changed to include the new phase parameter:
```
Fittable parameters: ('slope_0', 'intercept_0', 'amplitude_1', 'frequency_1', 'phase_1')
```